Documentation/guides/building-apps/build-properties.md

```diff
@@ -132,7 +132,7 @@ the Android [binutils][binutils] such as `ld`, the native linker,
 and `as`, the native assembler. These tools are included in the
 Xamarin.Android installation.
 
-The default value is `$(MonoAndroidBinDirectory)\binutils\`.
+The default value is `$(MonoAndroidBinDirectory)\binutils\bin\`.
```

Comment:
If we're using LLVM now and not binutils, why is `binutils` still in the directory name?